account_invoice_change_currency/models/account_change_currency.py

```diff
@@ -84,6 +84,8 @@ def _onchange_currency_change_rate(self):
         state = self.get_force_rate_state()
         if state.new_value_integer:
             return
+        if self._context.get('tracking_disable'):
+            return
```

Comment:
Yes, because in the tests I used
```
cls.env['account.invoice'].with_context({'tracking_disable': True})
```

Thus I made the corresponding fix in `account_invoice_change_currency`.